packages/apps/postgres/values.yaml

```diff
@@ -7,6 +7,7 @@
 ##
 external: false
 size: 10Gi
+walSize: 1Gi
```

Comment:
The name `walSize` may be confusing with `max_wal_size`, `min_wal_size` in `postgresql.conf`.

I'd go for `walStorageSize` here.